### DISCUSSION TITLE
Can't start on Windows 7

### DISCUSSION THREAD

**Discussion by ewerestr** [rustdesk/rustdesk#552] [Q&A] [answered]
I've build it on windows 10, tested it and everything's ok, but it can't start on Windows 7 and Windows Server 2008. I got messagebox with error "... api-ms-win-shcore-scaling-l1-1-1.dll is missing...". I put that dll in build folder, then program has started to throw 0xc000007b error. I can't realize where i've made mistake.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/549
